Update README.md

adamjvalentini opened the pull request:
Updated the ReadMe with initial changed. More to come!

adamjvalentini committed:
Update README.md

Updated the ReadMe with initial changed. More to come!

Winterequiem reviewed (changes requested):
Consider adding some of the following to game rules (I know number of players and losing condition is already specified, but maybe it would be clearer to list them out at the top):
- A game has 2 - 4 players
- Player starts the game with $1500
- Player loses the game when balance < $0
- Player can trade with other players between dice rolls
- Player rolls 2 dice
  - If a player rolls double thrice in a row, they go to jail
- Jail 
  - Player can still interact with other players (I.e trade, collect rent, etc) 
  - 3 turns to roll a double to get out
  - After three turns, must pay a $50 bail
- Buying a Property 
  - Each property has a buy price, mortgage value, and rent value
- Mortgage a Property 
  - Player receives the mortgage value
  - To unmortgage, pay 1.1x mortgage value.
  - Player cannot collect rent on a mortgaged property
- Build House/Hotel 
  - House and hotel have a build cost. 
  - To build a house player must own the full (colour) set
  - Houses must be built evenly. (I.e. Every property in the set must have 1 house built before player can build a second house)
  - Player can build a hotel after building 4 houses
- Land on Utility 
  - Pay 4x / 10x the number rolled by dice if owner owns 1 / 2 utilities
- Land on Railroad 
  - Pay $25 / $50 / $100 / $200 if owner owns 1 / 2 / 3 / 4 railroads

adamjvalentini committed:
Update README.md

Updated README file with requested changes. Thank you to Winterequiem for the suggestions.

Winterequiem commented on README.md:
```diff
@@ -1,39 +1,39 @@
+# Game Rules 
+The monopoly game we have implemented is a 2-4 player interactive gameboard program. Upon initialization of the game, users are prompted to role the dice of the program. After a player rolls a dice the following mutually exclusive events may occur: 
+-	A game has 2 - 4 players
+-	Player starts the game with $1500
```
A player starts the game with $1500 should mentioned before 'the following mutually exclusive events may occur.' 
'2 - 4 players' is already mentioned before this sentence as well, so it doesn't need to be in the list. 

Append the following to the list:
Land on Pass GO 
  - Collect $200
  
Land on Free Parking / Just Visiting (Jail)
  - Nothing happens
 
Land on Go To Jail 
  - Go to jail, don't pass go (I.e., Do not collect $200)
  
Land on Taxes 
  - Pay $200 or $100

adamjvalentini replied:
The edits have been added

adamjvalentini committed:
Update README.md

Edited ReadMe file according to review
